website/docs/d/elb_service_account.html.markdown

```diff
@@ -11,6 +11,8 @@ description: |-
 Use this data source to get the Account ID of the [AWS Elastic Load Balancing Service Account](http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#attach-bucket-policy)
 in a given region for the purpose of permitting in S3 bucket policy.
 
+~> **Note:** For AWS Regions opened since Jakarta (`ap-southeast-3`) in [December 2021](https://aws.amazon.com/blogs/aws/now-open-aws-asia-pacific-jakarta-region/), [AWS document](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#attach-bucket-policy) that [a service principal name](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_principal.html#principal-services) be used instead of AWS account ID in any relevant IAM policy.
```

Comment:
I don't think we need to link to the Jakarta blog post; I didn't see anything in the blog post that was immediately relevant to this particular document.

```suggestion
~> **Note:** For AWS Regions opened since Jakarta (`ap-southeast-3`) in December 2021, AWS [documents that](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-access-logs.html#attach-bucket-policy) a [service principal name](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_principal.html#principal-services) should be used instead of an AWS account ID in any relevant IAM policy.
```